vocab/credentials/v2/vocabulary.yml

```diff
@@ -45,56 +50,56 @@ class:
     label: Credential evidence
     comment: A Credential Evidence scheme provides enough information to a verifier to determine whether the evidence gathered meets their requirements for issuing a credential. The precise content of each evidence scheme is determined by the specific evidence type definition.
 
+  - id: CredentialGraph
+    label: (Verifiable) credential graph
+    comment: Instances of this class are RDF Graphs, where each of these graphs must include exactly one <a href="#Credential">Credential</a> (usually a <a href="#VerifiableCredential">Verifiable Credential</a>).
```

Comment:
I strongly dislike the "usually" language here...

Reply:
This is a bit the same question as for `Presentation` vs. `VerifiablePresentation`. Do we want to have a separate `CredentialGraph` and a `VerifiableCredentialGraph`? We do not know what the future holds, and I am tempted to define a `CredentialGraph` only, which is used to encapsulate a graph with a single `Credential`. Because a `VerifiableCredential` is also a `Credential`, this covers our current usage. Hence my choice in the naming.